cli/Makefile

```diff
@@ -4,6 +4,7 @@ SHELL := /bin/bash
 VERSION=v0.1.0
 BINARY=bin/cft
 GITHUB_REPO=github.com/GoogleCloudPlatform/cloud-foundation-toolkit
+GOOGLE_APPLICATION_CREDENTIALS=
```

Comment:
Is this an SA key? can't we get the credentials via gcloud auth login?

Reply:
removed to do not confuse users